app/basketball/[competition]/matches/match.css

```diff
@@ -47,4 +47,60 @@
     width: 100px; /* Adjust as needed */
   }
   
+  /* Style for the matchStat table */
+  .matchStat {
```

Comment:
prefer to use utility over explicit css. 
We are using tailwindcss under the hook, 
for example for padding, https://tailwindcss.com/docs/padding